content/en/docs/reference/tools/_index.md

```diff
@@ -8,10 +8,10 @@ no_list: true
 ---
 
 <!-- overview -->
-Kubernetes contains several built-in tools and external tools that are commonly used or relevant that may as well be seen as required for Kubernetes to function.
-
+Kubernetes contains several tools to help you work with the Kubernetes system.
 
 <!-- body -->
+{{% thirdparty-content %}}
 
 ## Minikube
```

Comment:
Minikube is not a third-party tool: it's part of the Kubernetes project.

Reply:
May be we can provide third party content short code just before Helm wdyt?

Reply:
Yes, but: the existing shortcode isn't right for an _item_. I can make a PR to update that shortcode so it is.

Reply:
I made #29877 with the aim of helping out here.